Add wake plugin (WoL wrapper with MAC + broadcast) (#703)
ohmybash/oh-my-bash: plugins/wake/README.md

# wakeonlan

This plugin provides a wrapper around the "wakeonlan" tool available from most

> distributions' package repositories, or from [the following website](https://github.com/jpoliv/wakeonlan).

Reviewer: ```suggestion
distributions' package repositories, or from [this website](https://github.com/jpoliv/wakeonlan).
```

It says "the following", but the link doesn't follow this text, but the text itself is linked.

To use it, add `wake` to the plugins array in your bashrc file:

```bash
plugins=(... wake)
```

## Usage

In order to use this wrapper, create the `~/.wakeonlan` directory, and place in
that directory one file for each device you would like to be able to wake. Give
the file a name that describes the device, such as its hostname. Each file
should contain a line with the mac address of the target device and the network
broadcast address.

For instance, there might be a file `~/.wakeonlan/server` with the following
contents:

```
00:11:22:33:44:55:66 192.168.0.255
```

To wake that device, use the following command:

```console
wake server

> ```

Reviewer: When `console` is specified as a language, the text not followed by `$` or `#` is considered to be outputs of commands. `wake server` is the command in the present case, so it should be prefixed by `$` or `#`. Or the language specification should be `sh` or `bash` instead of `console`.

The available device names will be autocompleted, so:

```console
wake <tab>
```

> ...will suggest "server", along with any other configuration files that were

Reviewer: ```suggestion
...will suggest "server", along with any other configuration files that are
```

placed in the `~/.wakeonlan` directory.

> You can also just type `wake` to list show usage and available devices:

Reviewer: ```suggestion
You can also just type `wake` to show usage and list available devices:
```

```
Usage: wake <device>

> Available devices: server 

Reviewer: ```suggestion
Available devices: server
```
